romancal/regtest/test_wfi_pipeline.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+
+from romancal.pipeline.exposure_pipeline import ExposurePipeline
+import roman_datamodels as rdm
+from romancal.assign_wcs.assign_wcs_step import AssignWcsStep
+import numpy as np
+from numpy.testing import assert_allclose
+from .regtestdata import compare_asdf
+import copy
+
+
+def passfail(bool_expr):
+    if bool_expr:
+        return "Pass"
+    else:
+        return "Fail"
+
+@pytest.mark.bigdata
+@pytest.mark.soctests
+def test_level2_image_processing_pipeline(rtdata, ignore_asdf_paths):
+    rtdata.get_data("WFI/image/l1_0001.asdf")
+    rtdata.input = "l1_0001.asdf"
+
+    # Test Pipeline
+    output = "l1_0001_cal.asdf"
+    rtdata.output = output
+    args = ["--disable-crds-steppars",
+            "--steps.jump.rejection_threshold=180.0",
+            "--steps.jump.three_group_rejection_threshold=185.0",
+            "roman_elp", rtdata.input,
```

Comment:
If you add 
--steps.jump.four_group_rejection_threshold=190.
This might speed it up more.

Reply:
Added, thanks.